sections/introduction.md

```diff
@@ -0,0 +1,49 @@
+# Introduction
+
+> ℹ️ These docs are written for engineers and ad ops professionals looking to integrate zero.js in their site.
+
+## Background
+
+The programmatic advertising system emits worrying amounts of CO2. In the worst cases, delivering a single impression involves the processing, storing and transmitting of data across hundreds of nodes. The result of this noisy system is that a single impression can emit over 1 gram of carbon dioxide 🤯
```

Comment:
To align to the commercial you could add this at the beginning: Digital advertising is responsible for 10% of all internet carbon emissions and the internet is responsible for 2% of global emissions.